coml/prompt_utils.py

```diff
@@ -459,6 +471,20 @@ def cached_generate_fewshots(prompt_version: str) -> list[GenerateContext]:
     with open(
         Path(__file__).parent / f"prompts/generate_fewshots_{prompt_version}.json"
     ) as f:
+        if prompt_version in ["matplotlib", "seaborn"]:
+            fewshots = json.load(f)
+            for shot in fewshots:
+                variables = {}
+                shot_codes = "\n".join(shot["codes"])
+                shot_codes = shot_codes.replace(
```

Comment:
Support arbitrary code execution in shot codes seem unnecessary. It also confuses the semantics of `codes`.

Suggest writing "dataset" names in `generate_fewshots_xxx.json`, and load the corresponding datasets here.